package.json

```diff
@@ -0,0 +1,146 @@
+{
+  "name": "web",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^1.2.30",
+    "@fortawesome/free-brands-svg-icons": "^5.14.0",
+    "@fortawesome/free-regular-svg-icons": "^5.14.0",
+    "@fortawesome/free-solid-svg-icons": "^5.14.0",
+    "@fortawesome/react-fontawesome": "^0.1.11",
+    "@onaio/connected-private-route": "^0.0.11",
+    "@onaio/connected-reducer-registry": "^0.0.3",
+    "@onaio/gatekeeper": "^0.1.0",
+    "@onaio/loaders": "^0.0.1",
+    "@onaio/redux-reducer-registry": "^0.0.9",
+    "@onaio/session-reducer": "^0.0.11",
+    "@onaio/utils": "^0.0.1",
+    "@opensrp/server-logout": "^0.0.1",
+    "@opensrp/server-service": "^0.0.4",
+    "@types/antd": "^1.0.0",
+    "antd": "^4.6.3",
+    "bootstrap": "^4.5.2",
+    "client-oauth2": "^4.3.3",
+    "connected-react-router": "^6.8.0",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
+    "react-helmet": "^6.1.0",
+    "react-redux": "^7.2.1",
+    "react-router": "^5.2.0",
+    "react-router-dom": "^5.2.0",
+    "react-scripts": "3.4.3",
+    "redux": "^4.0.5",
+    "redux-thunk": "^2.3.0",
+    "seamless-immutable": "^7.1.4"
+  },
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "TZ=UTC react-scripts test --env=jest-environment-jsdom-sixteen",
+    "eject": "react-scripts eject",
+    "lint": "eslint 'src/**/*.{ts,tsx}' --max-warnings 0",
+    "coveralls": "set CI=true && yarn test --runInBand --verbose --coverage=true --forceExit --detectOpenHandles && cat ./coverage/lcov.info | coveralls"
```

Comment:
Let us add the `[no-large-snapshots](https://github.com/jest-community/eslint-plugin-jest/blob/master/docs/rules/no-large-snapshots.md)` eslint rule, and additionally ensure that we have something like `yarn lint-snap` that runs on CircleCI to effectively auto-fail PRs that have large snapshots.

Reply:
@KipSigei looks like this is not added to CircleCI